packages/bun-types/globals.d.ts

```diff
@@ -3800,3 +3800,11 @@ interface PromiseConstructor {
     reject: (reason?: any) => void;
   };
 }
+
+interface Navigator {
+  readonly userAgent: string;
+  readonly platform: "MacIntel" | "Win32" | "Linux x86_64",
+  readonly hardwareConcurrency: number
```

Comment:
I think both of these lines should end with semicolon `;` not comma or empty

Reply:
Fixed